Project "Velvet Sortie" (Justice Loadouts, Template Revisions)

M3739 opened the pull request:
~~I HAVE TAKEN THE FIRST NAPKIN!~~
**_On today's episode of grassroots innovation, we are implementing the Justice department into the loadout proper._**

# Description

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

Description.

**This PR's goal is to implement and tune the Justice department's items into the load out system to allow for more defined control of character creation for engaging in the Justice role, along with a review, proofread, and correction of the paperwork templates.**

**This PR consists of two parts; the loadout, and the paperwork.**

# - Loadout

The load out now includes the Justice department as it's own tab, with the associated vanities that come with it. In addition to follow through with a TODO comment from DeltaV, the CJ now has their gavel and it's block in `pocket1` and `pocket2` respectively. I added the block, as the gavel is otherwise just a funny looking hammer without it. Their loadout has also been adjusted to no longer give the CJ robes, togue, or gloves, as there is now greater control over the loadout.

DeltaV gave the Attorney and the Prosecutor the Lawyer Stamp via a bag prototype known as `ClothingBackpackLawyerFilled` and it's variants. However, due to how we use the loadout system, this addition from them never sees the light of day. It is now in the `pocket2` of both the Attorney and the Prosecutor's starting gear. The attorney's security book in `pocket1` has been removed to make it in line with the Prosecutor's lack of such. According to a TODO comment, once a space law book is implemented, they will put it in instead, and I agree on this regard.

The CJ now properly appears on the manifest as a member of command.

This was accomplished by modifying the necessary loadout and job YAMLs, and their associated localization FTLs. Any new files or additions has been placed in the Floof folder where applicable to separate from upstream. If I missed any, or I overlooked the fact I can separate it better, please let me know.

# - Paperwork Templates

The templates, primarily the Security and Justice templates, have received a revision. This includes but not limited to error corrections, formatting issues (with one of them in particular), spelling mistakes, and some rewrites to enhance clarity, and give better formality to the documentation. "Head of Personal" has been corrected to "Head of Personnel" where as applicable, and instances of 'Court Clerk', 'Clerk', and 'Clerk of Courts' has been corrected to 'Clerk of Court'. Medical templates are now properly colored, instead of being logistics colored, and have also received a revision.

This is not all of the changes in this. But overall enhances clarity, fixes errors, and should make bureaucracy more bearable for us dabbling in it.

This was accomplished by modifying the associated YAML files in the Floof 'Paper' Folder.

---

# TODO

<!--
A list of everything you have to do before this PR is "complete"
You probably won't have to complete everything before merging but it's good to leave future references
-->

- [x] Justice has it's own Jobs tab
- [x] Attorney and Prosecutor tabs done
- [x] Clerk and Chief Justice tabs done
- [x] Uncategorized tab done
- [x] Discuss any additions, changes, and address concerns or corrections from the maintainers
- [x] Y E E T

---

<!--
This is default collapsed, readers click to expand it and see all your media
The PR media section can get very large at times, so this is a good way to keep it clean
The title is written using HTML tags
The title must be within the <summary> tags or you won't see it
-->

<details><summary><h1>Media</h1></summary>
<p>

![LawyerServiceGONE](https://github.com/user-attachments/assets/5415cf2f-65a4-46ba-bb9a-fdeb5fa6b105)
![Attorney](https://github.com/user-attachments/assets/4152c3a3-b31e-4e60-903a-77ece2b07f07)
![Clerk](https://github.com/user-attachments/assets/4516d845-6753-4628-bf76-53d02b28732f)
![ChiefJustice](https://github.com/user-attachments/assets/fb36afb1-c9e3-4005-96c1-480848a102d2)
![Prosecutor](https://github.com/user-attachments/assets/e8b19fb6-bf9e-4e0c-aaa4-d3001418c949)
![JobListings](https://github.com/user-attachments/assets/f855c82f-ae6a-43c4-9395-c6f703df2969)
![Manifest](https://github.com/user-attachments/assets/f48818af-ee69-4e17-8127-956a2ab17fc8)
_The following image below is to showcase the new starting loadout changes, primarily `pocket1` and `pocket2`._
- Chief Justice
![CJLoadout](https://github.com/user-attachments/assets/e0c0138c-4b91-4e36-9918-9cba670aa6af)
- Attorney
![Attorney](https://github.com/user-attachments/assets/e56c1466-8c26-47ec-93c1-a34f4136afde)
- Prosecutor
![Prosecutor](https://github.com/user-attachments/assets/9455f38b-b6ce-4a9d-8ccf-77c43d8f4bb7)

</p>
</details>

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl: M3739
- add: The Justice department now has it's own loadout tab under 'Jobs'.
- tweak: The Chief Justice now properly appears on the manifest as a member of command, and their starting gear has been adjusted to be more in line with their fellow Command members.
- tweak: The Attorney and the Prosecutor now has a lawyer stamp as part of their starting gear.
- tweak: The Lawyer loadout tab is no longer under Service, It is now under Justice as 'Attorney'.
- tweak: Some document templates have received a revision. No more "Head of Personal" as the header of every Justice paperwork template.

M3739 commented:
Development will halt until #574 concludes with the implementation of the V4 loadout system.

In the meantime, discussion is encouraged as to what would be included in the uncategorized, or job specific tabs.

I am also welcome to any feedback, issues, and comments on this PR so far.

Mnemotechnician commented:
> Development will halt until #574 concludes with the implementation of the V4 loadout system.

Loadouts V4 will really only add item coloring and renaming (which must be enabled on loadout entries), so you're free to work on it right now. Besides, I'm not sure I will finish the merge in any reasonable amount of time...

M3739 commented:
> > Development will halt until #574 concludes with the implementation of the V4 loadout system.
> 
> Loadouts V4 will really only add item coloring and renaming (which must be enabled on loadout entries), so you're free to work on it right now. Besides, I'm not sure I will finish the merge in any reasonable amount of time...

Acknowledged. Thank you for the heads up, I shall resume work on this PR, then. Glad to hear it's not disruptive to existing loadouts.

M3739 commented:
Paperwork in the Security and Justice folders have been proofread, corrected, and rewritten for clarification. "Head Of Personal" is now the proper "Head of Personnel" spelling where applicable. "Court Clerk", "Clerk" or "Clerk of Courts" has been corrected to just "Clerk of Court". There is only one court, and it's the Station Court, and we use "Clerk of Court" instead of "Court Clerk" as a formality and to carry the gravitas of that position.

As a note for future writers, when addressing a position, rather than a person, write it as "the Head of Personnel" rather than just "Head of Personnel".

In addition, Medical now has proper coloring, and got a proofread of it's own.

M3739 commented:
The executive decision has been made to remove the loadout option for the security and justice paperwork template folders in the uncategorized tab. I have decided to just deliver it as is.

Anyways, time to write up the technical explanation, snap some photos, and finally,

#  _Y E E T_

M3739 commented:
DeltaV apparently added the Lawyer stamp to the starting loadout bags for Lawyer and Prosecutor respectively, but due to how our loadout system is implemented, that addition rarely, if at all, saw the light of day. It has been properly implemented by putting it into `pocket2` for both Attorneys and Prosecutors. The book has been removed from Attorney until a proper space law book is added, just as the TODO on the Prosecutor YAML mentions.

Chief Justice now properly appears on the manifest. Turns out it is a fairly simple fix.

M3739 commented:
Alright.

Project Velvet Sortie is deemed completed on my end. Time for the maintainers to scrutinize my work and question my aptitude.

M3739 commented:
Hold.

Some documents need a few corrections. Standby.

M3739 commented:
Done. Now we are ready for review. Had to address a Catch 22 in a document after talking with peeps.

Memeji reviewed (approved):
Looks good content-wise. Just need a further look through the code and should be good to go.

Mnemotechnician reviewed (approved):
Don't see any issues either, so approving.
